src/app.js

```diff
@@ -176,7 +180,12 @@ router.use(setAxiosDefaults);
 
 router.use("/oauth2", commonExpress.routes.oauth2);
 
-router.use(APP.PATHS.CHECK, require("./app/check"));
```

Comment:
https://expressjs.com/en/guide/routing.html#express-router

> express.Router
> Use the express.Router class to create modular, mountable route handlers. A Router instance is a complete middleware and routing system; for this reason, it is often referred to as a “mini-app”.

The use of a separate router is an intentional choice, in order to keep the `app.js` as streamlined as possible. Note that there are only a number of `router.use` lines at the bottom of this file.